src/raygun.rum/index.js

```diff
@@ -1024,7 +1024,7 @@ var raygunRumFactory = function (window, $, Raygun) {
       if (window.raygunUserAgentData.platform === "Windows") {
         var platformVersion = (window.raygunUserAgentData.platformVersion || '.').split(".");
         var majorVersion = parseInt(platformVersion[0] || '0', 10);
-        if (majorVersion !== NaN && majorVersion !== Infinity && majorVersion >= 13) {
+        if (!isNaN(majorVersion) && majorVersion !== Infinity && majorVersion >= 13) {
```

Comment:
For this sort of thing I would usually do:

```
var majorVersion = parseInt(platformVersion[0], 10) || 0;

if (majorVersion >= 13) {
```

Since `parseInt` will return `NaN` for any input that's not coercible to a number. I don't think you need to worry about `parseInt` returning `Infinity`.

Not critical to address for this PR, more just general code style :)

Reply:
I guess if the product version was something weird then maybe we could get an infinite value. but even if we did it would just repot win 11 not win 10. So no big deal

Reply:
Nice. I'm not aware of any inputs that cause `parseInt` to return `Infinity`, and from what I see on mdn it either returns a number or NaN, so we should be okay :)